eval/src/tests/eval/reference_operations/reference_operations_test.cpp

```diff
@@ -10,14 +10,15 @@ using namespace vespalib;
 using namespace vespalib::eval;
 using vespalib::eval::test::GenSpec;
 
-TensorSpec dense_2d_some_cells(bool square) {
+TensorSpec dense_2d_input(bool square) {
     return TensorSpec("tensor(a[3],d[5])")
         .add({{"a", 1}, {"d", 2}}, square ? 9.0 : 3.0)
         .add({{"a", 2}, {"d", 4}}, square ? 16.0 : 4.0)
-        .add({{"a", 1}, {"d", 0}}, square ? 25.0 : 5.0);
+        .add({{"a", 1}, {"d", 0}}, square ? 25.0 : 5.0)
+        .normalize();
```

Comment:
should move normalize call to where it needs to be (as in not when used as input)